<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>614</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>73</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>165</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>73</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>165</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>52</red>
        <green>73</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>165</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>44</red>
        <green>62</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 62, 80);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>15</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
image: url(:/.png/files.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>27</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
gridline-color: rgb(44, 62, 80);
border-color: rgb(44, 62, 80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Böngéssz ki egy képet, vagy másold ide az elérési útvonalat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>15</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
image: url(:/.png/start.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tips">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>505</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scrollTips">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>511</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
image: url(:/.png/arrow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="scannedObject">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>27</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>  Miből főzzünk?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recipeTitle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>105</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="instructions">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>105</y>
      <width>341</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ingredients">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>155</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextRecipe">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>255</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
</string>
    </property>
    <property name="text">
     <string>Következő</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevRecipe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
</string>
    </property>
    <property name="text">
     <string>Előző</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showRecipe">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>15</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(66, 94, 121);
image: url(:/.png/3448102.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources/images/res.qrc"/>
 </resources>
 <connections/>
</ui>
